lib/statesman/adapters/active_record.rb

```diff
@@ -41,8 +41,8 @@ def create(from, to, metadata = {})
         @last_transition = nil
       end
 
-      def history
-        if transitions_for_parent.loaded?
+      def history(force_reload: false)
```

Comment:
Is this not slightly odd, in that it means that the interface of the ActiveRecord adapter differs from the other adapters? `#history` is a public part of the adapter interface, and the `Mongoid` and `Memory` adapters won't support this additional parameter. It might be good to expand this change to them too.

Reply:
I don't think so: each of the adapters currently has a different interface! If you're explicitly calling the adapter's methods then you should know what interface you have available for that adapter.

Reply:
I'm not convinced that's quite true - e.g. `Memory#last` seems to accept any argument, presumably for compatibility with other adapters:

https://github.com/gocardless/statesman/blob/b962ed219e6b05328694def8f9a1d5a504ec5c2e/lib/statesman/adapters/memory.rb#L31